ibis/impala/client.py

```diff
@@ -26,6 +26,8 @@
 import numpy as np
 import pandas as pd
 
+import turbodbc
```

Comment:
This will need a new entry into `setup.py`. You can add a new pip extra called `'odbc'`, like this:

```python
odbc_extras = ['turbodbc']

setup(
    ...
    'odbc': odbc_extras
    ...
)
```